### PR TITLE
Cleanup worker for challenge tables

### PR DESCRIPTION
**Pull request by Hitenjain14**
### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/blobber/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- 0chain:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  |   |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  |   |
| zwalletcli  |   |
| 0chain  |   |
| blobber  | feat/challenge-cleanup  |
| authorizer  |   |
| 0box  |   |
| 0dns  |   |

**Review by dabasov — APPROVED** [review]
lgtm

File: code/go/0chain.net/blobbercore/challenge/timing.go [hunk 1]
```diff
@@ -186,3 +187,24 @@ func GetChallengeTiming(challengeID string) (*ChallengeTiming, error) {
...
+				cleanUpTimingWorker()
+			}
+		}
+	}()
+}
+
+func cleanUpTimingWorker() {
+	_ = datastore.GetStore().WithNewTransaction(func(ctx context.Context) error {
+		db := datastore.GetStore().GetTransaction(ctx)
+		return db.Model(&ChallengeTiming{}).Unscoped().Delete(&ChallengeTiming{}, "created_at_blobber < ?", time.Now().Add(-cleanupGap).Unix()).Error
```
Comment (dabasov) [inline code/go/0chain.net/blobbercore/challenge/timing.go:208]: can we have cleanupGap in configs?

**Review by dabasov — CHANGES_REQUESTED** [review]

File: code/go/0chain.net/blobbercore/challenge/timing.go (hunk 1 above)
Comment (Hitenjain14) [reply code/go/0chain.net/blobbercore/challenge/timing.go:208]: Move the challenge one to config

**Review by dabasov — APPROVED** [review]
lgtm

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  |   |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  |   |
| zwalletcli  |   |
| 0chain  |   |
| blobber  | sprint-1.11  |
| authorizer  |   |
| 0box  |   |
| 0dns  |   |

**Review by dabasov — APPROVED** [review]
lgtm

**Pull request merged**